arches/app/utils/data_management/resources/formats/rdffile.py

```diff
@@ -429,8 +431,21 @@ def is_concept_node(self, uri):
                 return True
         return False
 
+    def find_local_references(self, jsonld_document):
+        if "@id" in jsonld_document and "@type" in jsonld_document:
```

Comment:
suggest a further check that it is NOT a tile reference, as tiles can only exist in one place in the tree.

Reply:
what would that check look like?  I'm unsure of how to check for that.

Reply:
I assume you mean references like these
```
"http://localhost:8000/tile/d83757c8-203e-4755-b1fb-45e260105d78/node/e2f83f46-bacf-11ea-81b2-3af9d3b32b71"
```

Reply:
To me, the added complexity of implementing a regex to filter out those tile references doesn't seem to offset the additional entries in the cache.  I think I'm just going to leave it as it for now.